packages/service/src/db.test.ts

```diff
@@ -59,3 +59,68 @@ test("crdts can be persisted", async () => {
     ydoc2.getText("codemirror").toJSON()
   );
 });
+
+test("document updates can be persisted", async () => {
+  // create a yjs document A
+  const ydoc = new Y.Doc();
+  const originalState = Y.encodeStateAsUpdate(ydoc);
+  // make a change to the document
+  const ytext = ydoc.getText("codemirror");
+
+  // create a document to associate updates with
+  const dbDoc = await db.document.create({
+    data: {
+      value: "test value",
+      origin: Buffer.from(originalState),
+      web_rtc_key: "",
+    },
+  });
+
+  // create and persist update encodings
+  ytext.insert(0, "hello world");
+  const updatedState1 = Y.encodeStateAsUpdate(
+    ydoc,
+    Y.encodeStateVectorFromUpdate(originalState)
+  );
+  await db.document_updates_queue.create({
+    data: {
+      document_id: dbDoc.id,
+      document_update: Buffer.from(updatedState1),
+    },
+  });
+  ytext.insert(0, "foo bar");
+  const updatedState2 = Y.encodeStateAsUpdate(
+    ydoc,
+    Y.encodeStateVectorFromUpdate(updatedState1)
+  );
+  await db.document_updates_queue.create({
+    data: {
+      document_id: dbDoc.id,
+      document_update: Buffer.from(updatedState2),
+    },
+  });
+  ydoc.destroy();
+
+  // get the updates back from the database
+  const updatesFromDatabase = await db.document_updates_queue.findMany({
+    where: {
+      document_id: dbDoc.id,
+    },
+  });
```

Comment:
@tucker-weed @tmsavage this might be a good opportunity to test out some more complicated queries, with your ORMs

Lets extract this to its own function, and define what the inputs and outputs are
```ts
function LoadDocument(document_id: string ): Doc {
  ...
}
```

The trick here is that we need to get information from both tables. We need the origin from the Documents table, and we need this set of updates from the Updates table.